.github/workflows/deploy.yml

```
@@ -0,0 +1,46 @@
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    # 1. Checkout code
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    # 2. Log in to Docker Hub
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)